firefox-ios/firefox-ios-tests/Tests/ClientTests/Microsurvey/MicrosurveyPromptStateTests.swift

```diff
@@ -65,6 +65,24 @@ final class MicrosurveyPromptStateTests: XCTestCase {
         XCTAssertEqual(newState.showPrompt, true)
     }
 
+    func testDefaultAction() {
+        let initialState = MicrosurveyPromptState(
+            windowUUID: .XCTestDefaultUUID,
+            showPrompt: true,
+            showSurvey: true,
+            model: MicrosurveyMock.model
+        )
+        let reducer = microsurveyReducer()
+
+        let action = getInvalidAction()
```

Comment:
since we're only using this call in one place, I would say we can use
`MicrosurveyPromptAction(windowUUID: .XCTestDefaultUUID, actionType: InvalidActionType.invalid)`
 
and move it out to a separate function if we're using it for at least 2+ places. It's easier for readability in this test and clear we are sending a fake action. 

What do you think about also maybe using `FakeActionType.testAction` instead since its not really invalid, but does not modify the microsurvey state.

Reply:
Absolutely! Naming is always the hardest part in development 😂 . I really like your suggestion and agree it makes sense to keep the declaration inline for readability, rather than creating a separate function for a single-use action.